distribution/docs/src/main/resources/content/_coreConcepts/cryptography-standards.adoc

```diff
@@ -0,0 +1,23 @@
+:type: subCoreConcept
+:status: published
+:title: Cryptography Standards
+:parent: Standards Supported by ${branding}
+:order: 04
+
+.Cryptography Standards Provided by ${ddf-branding}
+[cols="2,1" options="header"]
+|===
+|Standard
+|Support Status
+
+|https://tools.ietf.org/html/rfc5246[Transport Layer Security (TLS) v1.1 & v1.2]
+|Supported
+a|https://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html#SupportedCipherSuites[Cipher Suites]
+
+* TLS_DHE_RSA_WITH_AES_128_GCM_SHA256
```

Comment:
❓ should we link somewhere in where the configuration is for this? I believe this is configurable. Just a consideration may not add any value

Reply:
We _could_. My original rationale for not including was that the audience for this section is different from the person doing the actual installation